src/cli/mod.rs

```diff
@@ -222,6 +222,14 @@ impl Invocation {
             "CARGO_TARGET_ARMV5TE_AXIS_LINUX_GNUEABI_RUSTFLAGS=--cfg crossbeam_no_atomic_64",
         ]);
 
+        // Pass through some env vars to docker container that cargo might need to fetch dependencies
+        let pass_through_env = ["http_proxy", "https_proxy"];
+        for var in pass_through_env {
+            if let Ok(val) = std::env::var("http_proxy") {
```

Comment:
should probably check var(var)

Reply:
Hmm… checking various tools, I'm thinking that maybe we should pass through any/all upper and lowercase versions of:
```
[
  "http_proxy",
  "https_proxy",
  "ftp_proxy",
  "all_proxy",
  "no_proxy",
]
```

I have no idea which tools inside the build container will respect which variables with which precedence, but at least that way `cargo-acap` isn't making that decision.

Reply:
Agreed! Will do tomorrow!